src/wp-admin/edit-comments.php

```diff
@@ -329,6 +353,6 @@
 <div id="ajax-response"></div>
 
 <?php
-wp_comment_reply('-1', true, 'detail');
+wp_comment_reply( '-1', true, 'detail' );
 wp_comment_trashnotice();
-include( ABSPATH . 'wp-admin/admin-footer.php' ); ?>
```

Comment:
`include` changed to `require`.

Reply:
As per review meeting, this is fine to keep the change